pyproject.toml

```diff
@@ -55,7 +55,16 @@ autoclasstoc = "^1.2"
 [tool.poetry-dynamic-versioning]
 enable = true
 vcs = "git"
-style = "semver"
+style = "pep440"
```

Comment:
To help me understand this PR, could you elaborate on the motivation to the changes to the "style" and "format-jinja" options?

We have previously been using Semantic Versioning, so it would be useful to continue in the same style if possible. I don't know of any differences between "semver" and "pep440" that apply to our use case, as we're just using straightforward versions of the form "X.Y.Z", as described in the contributing guide.

From reading the documentation of poetry-dynamic-versioning, it looks like the `style` option defines a custom format, so I'm not clear why a bespoke format pattern would be necessary for our rather vanilla use case?

Reply:
I thought that moving to the PEP440 standard would be preferable, and I find it to be slightly clearer. To clarify, the only difference we will see is when developing software, the semver system produces versions like '1.7.1.post12xxx', wheres PEP440 gives '1.7.1.dev12'. Both versions are exactly the same for tagged releases, and I can revert this change if preferred.

Reply:
Happy with that, agreed it doesn't make much difference.
What about the "format-jinja" option, is that necessary?

Reply:
It clarifies the behaviour of the versioning library for PEP440, and provides us an easy way to change this behaviour if we desire to in the future.